Source/DiscordRpc/Public/DiscordRpcBlueprint.h

```diff
@@ -94,6 +99,10 @@ class DISCORDRPC_API UDiscordRpc : public UObject {
     GENERATED_BODY()
 
 public:
+
+    UFUNCTION(BlueprintPure)
+        int64 GetStartTimeStamp() const {return FDateTime::Now().ToUnixTimestamp();}
```

Comment:
Shouldn't `UtcNow()` instead of `Now()` be here?

Reply:
For creating a timestamp for time elapsed in a Discord Rich Presence using DiscordRPC, it's generally a good practice to use Coordinated Universal Time (UTC) to ensure consistency across different time zones. In this context, Datetime::UtcNow() would be more suitable.  This ensures that the timestamp reflects the same point in time regardless of the user's local time zone, making your Rich Presence display consistent for users worldwide.

Reply:
You are correct, this has been fixed in the most recent version in my repos